### PR TITLE
Interiorshadoweff to gauges

### PR DESCRIPTION
**Pull request by wlbragg**
I didn't notice anything to outstanding. I'm not sure it works exactly the way you would want . t almost appears that the shadow may not align the same from instrument to gauge to panel.
But for the flashlight you can see it works really well
flashlighgr
.

### CODE REVIEW

**Comment by wkitty42** [comment]
from what i see in the picture, it looks good... just need to be able to direct it or have it a little larger so we can see the necessary switches and levers... nice job, too!

File: Models/c172p.xml [hunk 1]
```diff
@@ -258,7 +258,7 @@
     <object-name>CarbHeat</object-name>
     <object-name>CarbHeatMount</object-name>
 	<object-name>GenSwitch</object-name>
-	<!--object-name>LandingLightSwitch</object-name-->
+	<object-name>LandingLightSwitch</object-name>
```
Comment (onox) [inline Models/c172p.xml]: What's the difference?

**Comment by onox** [comment]
Does this fix #246?

**Comment by onox** [comment]
And how do I test the flash light?

**Comment by wlbragg** [comment]
You don't test the flashlight until it is implemented in fgdata. But we need to make sure there are no negative side effects from the interior shadow effect on the gauges.

No differences in Landing light switch, i uncommented it so it takes the shadow effect.

**Comment by wlbragg** [comment]
Yes it fixes issue #246

**Pull request merged**